crypto/objects/objects.txt

```diff
@@ -1689,6 +1689,11 @@ secg-scheme 14 3 : dhSinglePass-cofactorDH-sha512kdf-scheme
 1 3 6 1 4 1 311 60 2 1 2	: jurisdictionST	: jurisdictionStateOrProvinceName
 1 3 6 1 4 1 311 60 2 1 3	: jurisdictionC		: jurisdictionCountryName
 
+# RFC 9106 Argon2 (see https://www.rfc-editor.org/rfc/rfc9106.txt)
+                            : ARGON2I  : argon2i
+                            : ARGON2D  : argon2d
+                            : ARGON2ID : argon2id
+
```

Comment:
If I understand it right the Argon2 NIDs aren't used. Could you please remove them? As there do not seem to be any official Argon2 object identifiers, there is no point in adding it to the object database.